module/Photo/Module.php

```diff
@@ -105,4 +125,24 @@ public function getServiceConfig()
         );
     }
 
+    /**
+     * Get view helper configuration.
+     *
+     * @return array
+     */
+    public function getViewHelperConfig()
+    {
+        return array(
+            'factories' => array(
+                'isAllowed' => function ($sm) {
+                    $locator = $sm->getServiceLocator();
+                    $helper = new \Application\View\Helper\Acl();
```

Comment:
Since this is an `Application` view helper, it makes more sense to include it in the view helper configuration there.